packages/document/main-doc/docs/en/guides/basic-features/html.mdx

```diff
@@ -4,13 +4,13 @@ sidebar_position: 9
 ---
 # HTML Template
 
-Modern.js provides **JSX syntax** and **HTML(EJS) syntax** for customizing HTML template.
+Modern.js provides **JSX syntax** and **HTML(EJS) syntax** to customize the HTML template.
 
-## JSX syntax
+## JSX Syntax
 
-Modern.js convention, in the `src/`, or in the entry directory, you can create `Document.[jt]sx` and export a component by default. The rendering result of this component can be used as an HTML template for the entry.
+According to Modern.js conventions, you can create a `Document.[jt]sx` file under `src/` or the entry directory and export the component by default. The rendering result of this component can be used as the HTML template of the entry.
```

Comment:
"export the component by default" like 「default behavior」，Is it possible use "Default export a component" ?